protocol/README.md

```diff
@@ -25,147 +25,149 @@ Every TBDex message contains the following fields:
 ```json
 {
-  "credentialsSubmission": {...},
-  "acceptedQuoteId": "5678",
-  "deliveryInstructions": "leave a doggy bag with cheese on address 1234"
+    "orderId": 32432,
+    "quoteId": 2,
+    "paymentVerifiablePresentation": { ...}
 }
 ```
 
-## `PaymentRequest`
-
-| field    | data type | required | description                                                       |
-| -------- | --------- | -------- | ----------------------------------------------------------------- |
-| `paymentInstructions` | JSON Object    | Y        | Standard instructions of how the user should pay the PFI. |
-
+## `OrderStatus`
 
-## `PaymentReceipt`
-
-| field | data type | required | description |
-| ----- | --------- | -------- | ----------- |
-| `verifiableCredentialJwt` | string | Y | A VC that represents a receipt of payment in a JWT representation. |
-
-## `Close`
-
-| field    | data type | required | description        |
-| -------- | --------- | -------- | ------------------ |
-| `reason` | string    | ?        | Reason for closing |
-
-# State Machine Diagram
-
-A sequence of associated messages is defined as a message thread. This diagram illustrates all possible state sequences for a message thread.
-Each vertex represents a message type. Each edge represents who can transition the state of a message thread to the next vertex.
-
-For example, starting from the top: "A PFI can reply to an `RFQ` with a `Quote`"
-
-_Note: Assume that any vertex can transition to a `Close` by either participant_
+| field             | data type   | required | description                                                                             |
+| ----------------- | ----------- | -------- | --------------------------------------------------------------------------------------- |
+| `orderId`   | string | Y        | A unique identifier for the Order that was submitted |
+| `orderStatus` | enum      | Y        | Current status of Order that's being executed |
 
-```mermaid
-flowchart TD
-    accTitle: State Machine of Message Thread
-    accDescr: Possible state sequences for a message thread
-    start --> |Alice| RFQ
-    RFQ --> |PFI| Quote
-    Quote --> |Alice| Accept
-    Accept --> |PFI| PaymentRequest
-    PaymentRequest --> |Alice| MakePayment
-    MakePayment --> |PFI| PaymentReceipt
+PFI -> Alice: "Here's the status of your order."
+```json
+{
+    "orderId": 32432,
+    "orderStatus": "PENDING"
+}
 ```
 
 # Sequence Diagram
 
 ```mermaid
 sequenceDiagram
   actor Alice
-  actor PFI
-  Alice->>PFI: RequestForQuote
-  PFI->>Alice: []Quote
-  Alice->>PFI: Accept
-  PFI->>Alice: PaymentRequest
-  Alice->>Interwebs: PayMoneyz
-  Interwebs->>PFI: PaymentHook
-  PFI->>Alice: PaymentReceipt
-  
+	actor PFI1
+  actor PFI2
+	actor PFIn
+	PFI1->>Alice: Offering
+	PFI2->>Alice: Offering
+	PFIn->>Alice: Offering
+  Alice->>PFI1: RequestForQuote (RFQ)
+  PFI1->>Alice: Quote
+  Alice->>PFI1: Order
+  PFI1->>Alice: OrderStatus
```

Comment:
`OrderStatus` would represent the status at a particular moment in time. How do status updates get reported?

Reply:
that is a great question for folks thinking more about messaging at the protocol level - i feel like i'm remembering someone mentioning webhooks

i suppose in a web2 world, Alice would ping PFI for GET/ OrderStatus endpoint, passing in orderId... but i think that's no longer the case in the web5 / DWN world.

would appreciate thoughts from @mistermoe @KendallWeihe

Reply:
I would suggest that we think more as an event driven architecture. I would imagine Alice listening on messages that get written to her DWN